HelpSource/Classes/String.schelp

```diff
@@ -676,13 +676,29 @@ Despite the fact that the method is called 'unixCmd', strong::it does work in Wi
 ::
 
 method::unixCmd
-Execute a UNIX command strong::asynchronously:: using the standard shell (sh).
+Execute an operating system command strong::asynchronously:: using the standard shell (code::sh:: on *nix, code::cmd:: on Windows).
+
+If you want to run a command without a shell, please use link::Classes/SequenceableCollection#-unixCmd::.
 argument:: action
 A link::Classes/Function:: that is called when the process has exited. It is passed two arguments: the exit code and pid of the exited process.
 argument:: postOutput
 A link::Classes/Boolean:: that controls whether or not the output of the process is displayed in the post window.
 returns::
-An link::Classes/Integer:: - the pid of the newly created process. Use link::Classes/Integer#-pidRunning:: to test if a process is alive.
+An link::Classes/Integer:: - the pid of the strong::shell:: running the command, not of the command itself (see note below for exceptions).
+
+To get the actual pid of the command, use link::Classes/SequenceableCollection#-unixCmd::, which does not use a shell and returns the actual pid of the command.
+
+Use link::Classes/Integer#-pidRunning:: to test if a process is alive.
+note::
+While the underlying system call will obtaing the pid of the shell, we may get an actual pid of the command in the following situations:
+list::
+## On *nix systems, if the command is run with code::exec::, e.g. code::"exec scsynth -u 57110".unixCmd;::, which causes the calling process (the shell) to be replaced with the new command
+## On macOS specifically, if the command consists of a single executable, where macOS seems the optimize the call by not spawining a new shell
+::
+For these reasons it might have appeared that we are always getting the pid of the actual command, as opposed to the shell, which is what the documentation erroneously stated in the past.
```

Comment:
I find this sentence a bit confusing, is there a way to state it more directly? Perhaps:

"Past documentation incorrectly stated that this method always returned the pid of the actual command."

Reply:
Thanks! I used your wording, it's much better.
I also added interpunction at the end of the two list entries, is that okay?